pages/index.js

```diff
@@ -71,19 +73,50 @@ export default function Home() {
     setTask(updatedTaskList);
   }
 
-  return (
-    <Wrap>
-      <WrapMask>
+  function appendTask(title, durationHour, durationMinute) {
+    const durationHourInput = durationHour === "0" ? "" : durationHour + "h";
+    const durationMinuteInput =
+      durationMinute === "0" ? "" : durationMinute + "min";
+    const newTasks = [
+      ...task,
+      {
+        id: generateID(),
+        taskName: title,
+        duration: durationHourInput + durationMinuteInput,
+        checked: false,
+      },
+    ];
+    setTask(newTasks);
+  }
+
+  function HomePage() {
+    return (
+      <>
         <Welcome />
         <DateStyle>{getCurrentDate()}</DateStyle>
         <TaskCompleted task={task} handleToggleTask={handleToggleTask} />
-        <Task task={task} handleToggleTask={handleToggleTask} />
+        <Task
+          task={task}
+          handleToggleTask={handleToggleTask}
+          setPage={setPage}
+        />
+      </>
+    );
+  }
```

Comment:
is this a component inside another component call Home? If so you need to change this layout, you can remove "homepage and set the return value of this inner function as the return value of the outer function.